src/api-explorer/v4-0/ConcurRequest.swagger2.json

```diff
@@ -0,0 +1,2562 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "Request as a Service",
+    "version": "4.0",
+    "title": "Concur Request API Documentation"
+  },
+  "host": "www.concursolutions.com",
+  "basePath": "/travelrequest/v4",
+  "tags": [
+    {
+      "name": "Request Resource",
+      "description": "Manage Request which is representing the starting point of pre-spend authorizations within Concur Request"
+    },
+    {
+      "name": "Workflow Resource",
+      "description": "Manage workflow transitions for a Request"
+    },
+    {
+      "name": "Travel Agency Resource",
+      "description": "Get details of Travel Agencies integrated with Concur Request"
+    },
+    {
+      "name": "Policy Resource",
+      "description": "Get available Request Policies"
+    },
+    {
+      "name": "Expected Expense Resource",
+      "description": "Manage expected expenses attached to a Request"
+    }
+  ],
+  "paths": {
+    "/requests": {
+      "get": {
+        "tags": [
+          "Request Resource"
+        ],
+        "summary": "Get the list of existing Requests",
+        "operationId": "getListUsingGET",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "view",
+            "in": "query",
+            "description": "Name of the perimeter defining the Requests to get",
```

Comment:
@quancheng-zhao 
```suggestion
            "description": "Names of the parameters defining the Requests to get",
```

Reply:
@MarcBell-SAP 

Perimeter is the good term I think, maybe there is a better term like "scope" or "filter". The idea is that the view name is associated to a a group of request having some properties in common.